package.json

```diff
@@ -33,17 +33,17 @@
     "package.json": "sort-package-json"
   },
   "dependencies": {
-    "isolmate": "github:defi-wonderland/isolmate#59e1804"
+    "solmate": "github:transmissions11/solmate#a9e3ea2"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.0.3",
-    "@commitlint/config-conventional": "17.0.3",
-    "@defi-wonderland/natspec-smells": "1.0.3",
+    "@commitlint/cli": "19.3.0",
+    "@commitlint/config-conventional": "19.2.2",
+    "@defi-wonderland/natspec-smells": "1.1.1",
     "ds-test": "github:dapphub/ds-test#e282159",
```

Comment:
I think we should remove `ds-test` completely, isn't it part of forge-std ?

Reply:
if we remove it then `forge-std` fails to compile ⚰️